src-electron/db/query-bitmap.js

```diff
@@ -87,8 +87,8 @@ SELECT
   BITMAP.SIZE AS SIZE
 FROM BITMAP
 INNER JOIN DATA_TYPE ON BITMAP.BITMAP_ID = DATA_TYPE.DATA_TYPE_ID
-WHERE DATA_TYPE.NAME = ? AND DATA_TYPE.PACKAGE_REF = ?`,
-      [name, packageId]
+WHERE (DATA_TYPE.NAME = ? OR DATA_TYPE.NAME = ?) AND DATA_TYPE.PACKAGE_REF = ?`,
+      [name, name.toLowerCase(), packageId]
```

Comment:
Are both the input arg 'name' and the column attribute DATA_TYPE.NAME fields case-insensitive?
If 'name' is only lower cased but nam DATA_TYPE.NAME has some weird capitalization, they might not necessarily match.
Using `COLLATE NOCASE` in the SQL query would avoid that issue.

Reply:
So I thought about this. I used like to overcome this at one point but didn't want to promote this so just kept it to either small/uppercase. None of the other variations.